test/test-parser.rb

```diff
@@ -121,4 +121,48 @@ def test_failure
       assert_equal([-22], statistics.collect(&:return_code))
     end
   end
+
+  class FormatCompatibilityTest < self
+    class DynamicColumnsTest < self
+      def test_labeled_columns_v0
```

Comment:
We don't use "vX" for the format. So please don't use the name. Please use description such as "no columns" instead of "vX".